tests/shared/test_arguments.py

```diff
@@ -0,0 +1,76 @@
+import sys
+from argparse import ArgumentParser
+
+import pytest
+
+from pubtools._pulp.arguments import SplitAndExtend
+
+FAKE_OPTION_VALUES = ["value1", "value2", "value3"]
+
+
+@pytest.fixture
```

Comment:
The usage of fixture here is a bit odd - half the tests are using it, but half the tests are just creating an ArgumentParser directly which is exactly what the fixture does.

If this was because some of the tests are looping over delimiter values and they want a new parser at each iteration - my suggestion below to use a fixture or pytest.mark.parametrize for the delimiters ought to fix that as the tests wouldn't need a loop any more.  Then you'd be able to use this fixture consistently for all tests.

Reply:
Resolved through use of parametrization as suggested in your other comment.